homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -718,22 +719,28 @@ def get_entity_state(config: Config, entity: State) -> dict[str, Any]:
     return data
 
 
-def entity_to_json(config: Config, entity: State) -> dict[str, Any]:
+@lru_cache(maxsize=1024)
+def _entity_unique_id(entity_id: str) -> str:
+    """Return the emulated_hue unique id for the entity_id."""
+    unique_id = hashlib.md5(entity_id.encode()).hexdigest()
+    return f"00:{unique_id[0:2]}:{unique_id[2:4]}:{unique_id[4:6]}:{unique_id[6:8]}:{unique_id[8:10]}:{unique_id[10:12]}:{unique_id[12:14]}-{unique_id[14:16]}"
```

Comment:
We should use the unique id if we have it like we do in homekit but that would be a breaking change

Reply:
Please break long strings around 88 characters per line.

Reply:
77a348a3bd0f53a373a1a63e9d1a0f5933e09864